src/client/Microsoft.Identity.Client/Platforms/iOS/iOSTokenCacheAccessor.cs

```diff
@@ -147,6 +146,27 @@ public IEnumerable<MsalAccountCacheItem> GetAllAccounts()
                 .ToList();
         }
 
+        internal SecStatusCode TryGetBrokerApplicationToken(string clientId, out string appToken)
+        {
+            var queryRecord = new SecRecord(SecKind.GenericPassword)
+            {
+                AccessGroup = _keychainGroup,
+                Account = clientId,
+                Service = iOSBrokerConstants.iOSBroker,
+            };
+
+            SecRecord record = SecKeyChain.QueryAsRecord(queryRecord, out SecStatusCode resultCode);
```

Comment:
nit: why not secStatusCode :)

Reply:
because

---
In reply to: [339276501](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1475#discussion_r339276501) [](ancestors = 339276501)